$NetBSD$

* Add node-ovsx-sign for Open VSX, the open-source alternative to Microsoft’s Visual Studio Code Marketplace

--- package.json.orig	2026-10-14 09:52:01.000000000 +0000
+++ package.json
@@ -156,6 +156,7 @@
     "native-is-elevated": "0.9.0",
     "native-keymap": "^3.3.5",
     "node-addon-api": "^6.0.0",
+    "node-ovsx-sign": "^1.2.0",
     "node-pty": "^1.2.0-beta.15",
     "open": "^10.1.2",
     "playwright-core": "1.61.0-alpha-2026-06-04",
